geniushubclient/zoneclasses/properties.py

```diff
@@ -0,0 +1,40 @@
+from geniushubclient.const import (
+    ITYPE_TO_TYPE,
+    ZONE_KIT,
+    ZONE_TYPE,
+)
+
+
+class Properties:
+    """The properties of the zone.
+
+        Intended to be the properties that are fixed for the lifecycle of the zone
+    """
+
+    def __init__(self, id):
+        self.id = id
+        self.name = None
+        self.zone_type = None
+        self.subtype = None
+        self.type_name = None
+        self.has_room_sensor = None
+
+    def update(self, json_data):
```

Comment:
If you're interested in typing, the convention here would be:
```
from typing import Dict

def update(self, json_data: Dict) -> None:
```

You don't do a `-> None` for the init dunder.

Reply:
Ta, done!